frontend/tests/e2e/support/commands.js

```diff
@@ -65,3 +65,149 @@ Cypress.Commands.add(
     cy.get('[data-cy="saveButton"]').click();
   }
 );
+
+Cypress.Commands.add('seeTournamentsLists', type => {
+  cy.get('[data-cy="Tournament"]').click();
+  cy.get(`[data-cy="${type}"]`).click();
+  cy.wait(100);
```

Comment:
Instead of waiting "randomly" it would be much better to wait for the actual response from the server.
Check [this link ](https://docs.cypress.io/guides/references/best-practices.html#Unnecessary-Waiting)for more info, any question or if you disagree let me know :D

Reply:
This applies to all waiting comments, you can probably have a similar approach in all